aws/resource_aws_sagemaker_training_job_test.go

```diff
@@ -0,0 +1,465 @@
+package aws
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/sagemaker"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+const sagemakerTestAccSagemakerTrainingJobResourceNamePrefix = "terraform-testacc-"
+
+func init() {
```

Comment:
It looks like this sweeper is unfinished and unneeded since we can't delete the training job. Do you mind removing this?